widget/view.html

```diff
@@ -194,46 +182,39 @@ <h6 class="font-weight-normal margin-bottom-sm modal-title text-transform-none">
-                                        class="padding-5 queue-card-header-text margin-right-3 font-size-14">
-                                        {{connector.label}}
-                                    </div>
-                                    <div
-                                        class="font-size-14 text-center margin-top-sm margin-bottom-sm justify-content margin-left-auto margin-right-sm">
-                                        <i data-ng-if="!connector.installed && connector.selectConnector"
-                                            class="text-color-warning icon icon-installing-lodder-static"></i>
-                                        <span data-ng-if="!connector.installed && connector.selectConnector"
-                                            style="color: #D2D5DA;">Installing...</span>
-                                        <i data-ng-if="connector.installed && connector.selectConnector"
-                                            class="text-success icon icon-check"></i>
-                                        <span data-ng-if="connector.installed && connector.selectConnector"
-                                            style="color: #D2D5DA;">Installed</span>
-                                        <i data-ng-if="!connector.installed && !connector.selectConnector"
-                                            class="text-danger icon icon-failed"></i>
-                                        <span data-ng-if="!connector.installed && !connector.selectConnector"
-                                            style="color: #D2D5DA;">Installed Failed</span>
-                                    </div>
-                                </div>
-                            </div>
+                        <div
+                            class="font-size-14 text-center margin-top-sm margin-bottom-sm justify-content margin-left-auto margin-right-sm">
+                            <i data-ng-if="!connector.installed && connector.selectConnector"
+                                class="text-color-warning icon icon-installing-lodder-static"></i>
+                            <span data-ng-if="!connector.installed && connector.selectConnector"
+                                style="color: #D2D5DA;">Installing...</span>
+                            <i data-ng-if="connector.installed && connector.selectConnector"
+                                class="text-success icon icon-check"></i>
+                            <span data-ng-if="connector.installed && connector.selectConnector"
+                                style="color: #D2D5DA;">Installed</span>
+                            <i data-ng-if="!connector.installed && !connector.selectConnector"
+                                class="text-danger icon icon-failed"></i>
+                            <span data-ng-if="!connector.installed && !connector.selectConnector"
+                                style="color: #D2D5DA;">Installed Failed</span>
```

Comment:
static color codes work for all themes ?

Reply:
removed unwanted color code and tested on all themes